.github/workflows/verification-tests.yml

```diff
@@ -0,0 +1,86 @@
+---
+name: Verification Tests
+
+env:
+  IMAGE_NAME: job-server
+  PUBLIC_IMAGE_NAME: ghcr.io/opensafely-core/job-server
+  REGISTRY: ghcr.io
+  SSH_AUTH_SOCK: /tmp/agent.sock
+
+on:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    - cron:  "47 0 * * *"
+
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+
+    services:
+      postgres:
```

Comment:
Technically, these tests don't require the use of a database. However, we've created a fixture called `enable_db_access_for_all_tests` and set `autouse=True` on it, which means that currently every test does require the database. This hasn't been an issue before, but it now seems unnecessary.

We can make that fixture optional (according to stackoverflow) by creating a pytestmark called `disable_db` and changing the fixture to this:

```python
@pytest.fixture(autouse=True)
def enable_db_access_for_all_tests(request):
    disable_db = request.node.get_closest_marker("disable_db")
    if disable_db:
        yield
    else:
        yield request.getfixturevalue("db")
```

Reply:
Thanks - that looks like it's worked :)